src/components/KToaster/KToaster.vue

```diff
@@ -11,33 +11,36 @@
       :class="`${toaster.appearance}`"
       role="alert"
     >
-      <div class="toaster-header">
-        <div class="toaster-icon-container">
-          <component
-            :is="getToastIcon(toaster.appearance)"
-            class="toaster-icon"
-            :color="KUI_COLOR_TEXT"
-          />
-        </div>
-        <span class="toaster-title">
+      <div class="toaster-icon-container">
+        <component
+          :is="getToastIcon(toaster.appearance)"
+          class="toaster-icon"
+          :color="KUI_COLOR_TEXT"
+        />
+      </div>
+      <div class="toaster-content">
+        <span
+          v-if="toaster.title"
+          class="toaster-title"
+        >
           {{ toaster.title }}
         </span>
-        <CloseIcon
-          class="toaster-close-icon"
-          :color="KUI_COLOR_TEXT_NEUTRAL_WEAK"
-          data-testid="toaster-close-icon"
-          role="button"
-          :size="KUI_ICON_SIZE_50"
-          tabindex="0"
-          @click="() => $emit('close', toaster.key)"
-        />
+        <p
+          v-if="toaster.message"
+          class="toaster-message"
+        >
+          {{ toaster.message }}
+        </p>
       </div>
-      <p
-        v-if="toaster.message"
-        class="toaster-message"
-      >
-        {{ toaster.message }}
-      </p>
+      <CloseIcon
+        class="toaster-close-icon"
+        :color="KUI_COLOR_TEXT_NEUTRAL_WEAK"
+        data-testid="toaster-close-icon"
+        role="button"
+        :size="KUI_ICON_SIZE_50"
+        tabindex="0"
+        @click="() => $emit('close', toaster.key)"
```

Comment:
question: why `@click="() => $emit('close', toaster.key)"` and not `@click="$emit('close', toaster.key)"` ?

Reply:
Updated